src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -134,20 +130,7 @@ export default defineComponent({
     },
     addCourse() {
       if (this.selectedCourse == null) return;
-      this.$emit('add-course', this.selectedCourse, {
-        optOut: getRelatedRequirementIdsForCourseOptOut(
-          this.selectedCourse.crseId,
-          this.selectedRequirementID,
-          store.state.groupedRequirementFulfillmentReport,
-          store.state.toggleableRequirementChoices,
-          store.state.userRequirementsMap
-        ),
-        // Only include the selected requirement from opt-in.
-        acknowledgedCheckerWarningOptIn: this.selfCheckRequirements
-          .filter(it => it.id === this.selectedRequirementID)
-          .map(it => it.id),
-        arbitraryOptIn: {},
-      });
```

Comment:
This chunk of code in `NewCourseModal.vue` basically ensured that there was at most one selectable requirement connected to the new course. This guard is no longer necessary, since each course can be connected to multiple selectable requirements. Instead, we should add the selectable requirement (if defined), which I defer to the parent since that's where the uniqueID of the course is determined.